engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1827,6 +1848,63 @@ private void orchestrateStop(final String vmUuid, final boolean cleanUpEvenIfUna
         advanceStop(vm, cleanUpEvenIfUnableToStop);
     }
 
+    private void getPersistenceMap(Map<String, Boolean> vlanToPersistenceMap, NetworkVO networkVO) {
```

Comment:
`getPersistenceMap` here doesn't return anything, _updatePersistenceMap_ might be best fit here